src/app_errors/mod.rs

```diff
@@ -250,12 +331,17 @@ impl From<Context<AppErrorKind>> for AppError {
 /// Generate HTTP error responses for AppErrors
 impl<'r> Responder<'r> for AppError {
     fn respond_to(self, request: &Request) -> response::Result<'r> {
+        match request.guard::<State<MozlogLogger>>() {
+            Outcome::Success(logger) => {
+                let log = MozlogLogger::with_app_error(&logger, &self)
+                    .map_err(|_| Status::InternalServerError)?;
+                slog_error!(log, "{}", "Request errored.");
+            }
+            _ => println!("Internal error: No managed MozlogLogger"),
```

Comment:
How come this isn't a  `slog_error!`?

Reply:
Duh, because haven't got a logger you div.

In that case, should we panic here? If logging isn't working we probably want to abort don't we?

Reply:
Yes, panicking sounds reasonable.